src/clients/c++/library/request_http.cc

```diff
@@ -238,30 +262,49 @@ class ServerStatusHttpContextImpl : public ServerStatusContext {
 
 ServerStatusHttpContextImpl::ServerStatusHttpContextImpl(
     const std::string& url, bool verbose)
+#if TRTIS_ENABLE_HTTP_V2
+    : url_(url + "/" + kHttpV2RESTEndpoint), verbose_(verbose)
+#else
     : url_(url + "/" + kStatusRESTEndpoint), verbose_(verbose)
+#endif
 {
 }
 
 ServerStatusHttpContextImpl::ServerStatusHttpContextImpl(
     const std::string& url, const std::map<std::string, std::string>& headers,
     bool verbose)
+#if TRTIS_ENABLE_HTTP_V2
```

Comment:
In V2 server status is only available for individual models, not for entire server, so this constructor should not exists / return error.

Reply:
Removed constructor, returning appropriate error in respective Create functions